root/components/Header.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * @flow
+ * Copyright (C) 2017 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const EntityTabs = require('./EntityTabs');
+const SubHeader = require('./SubHeader');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+const TaggerIcon = require('../static/scripts/common/components/TaggerIcon');
+
+type Props = {
+  entity: EntityT;
+  editTab?: string | React.Node;
+  headerClass: string;
+  heading?: string | React.Node;
+  page: string;
+  subHeading: string;
+};
+
+const Header = ({entity, headerClass, subHeading, ...props}: Props) => {
+  let heading = <EntityLink entity={entity} />;
+
+  if (props.heading) {
+    heading = props.heading;
+  }
+
+  const header = (
+    <div className={headerClass}>
+      {$c.session.tport ? <TaggerIcon entity={entity} /> : null}
```

Comment:
I think `$c.session` is sometimes null (perhaps if you're not logged in). So you'd have to do `{($c.sessionid && $c.session.tport) ? ...}`.